history/cultures/frostborn.txt

```diff
@@ -50,5 +48,7 @@
 	discover_innovation = innovation_armilary_sphere
 	discover_innovation = innovation_manorialism
 	#
+	discover_innovation = innovation_gryphon
```

Comment:
It raises the question of making gryphons regional to the Eastern Kingdoms. They don't use gryphons because they don't live in Northrend.

Reply:
![image](https://user-images.githubusercontent.com/74790101/138536222-2dc3b7b2-c6e0-467f-bc7b-f3183223757f.png)

I would agree making them regional is smart, however, I would still give it to the Northrend humans, but only them.  Meaning I would not include their territory as a requirement, but because there is lore about Northrend gryphons we can make an exception for them I think.